appinterface/projection/crossfire/crossfire.go

```diff
@@ -538,6 +560,59 @@ func (projection *Crossfire) updateTxSentCount(
 	return nil
 }
 
+func (projection *Crossfire) computeCommitmentRank(
+	rankKey string,
+	crossfireValidatorStatsPrefix string,
+	crossfireValidatorStatsView *view.CrossfireValidatorsStats,
+	crossfireValidatorView *view.CrossfireValidators,
+) error {
+	participants, err := projection.Client.Participants()
+	if err != nil {
+		return fmt.Errorf("error getting participants list %v", err)
+	}
+
+	participantsMap := make(map[string]bool)
```

Comment:
Explanation: create a participantsMap as { opArr: true } to filter the validatorCommit list in `validator_stats` table